mm/2s2h/Enhancements/Minigames/BeaverRace.cpp

```diff
@@ -60,4 +64,32 @@ void RegisterBeaverRace() {
     });
 }
 
-static RegisterShipInitFunc initFunc(RegisterBeaverRace, { CVAR_NAME });
+void RegisterBeaverRaceSpeedup() {
+    COND_ID_HOOK(ShouldActorUpdate, ACTOR_EN_AZ, CVAR_SPEEDUP, [](Actor* actor, bool* should) {
+        if (!CHECK_WEEKEVENTREG(WEEKEVENTREG_24_04)) {
+            SET_WEEKEVENTREG(WEEKEVENTREG_24_04);
+        }
+    });
+
+    COND_ID_HOOK(OnOpenText, 0x10D6, CVAR_SPEEDUP, [](u16* textId, bool* loadFromMessageTable) {
+        std::string replaceMsg = "My older brother will show you\x11the way, so follow him and\x11"
+                                 "don't get separated!";
+        auto entry = CustomMessage::LoadVanillaMessageTableEntry(*textId);
+        entry.msg.replace(entry.msg.begin(), entry.msg.end(), replaceMsg);
+
+        CustomMessage::LoadCustomMessageIntoFont(entry);
+        *loadFromMessageTable = false;
```

Comment:
Any particular reason you're using `entry.msg.replace` to entirely change the text, as opposed to the `entry.msg = "blah blah"` pattern that OnOpenText typically uses? e.g.:

```suggestion
        auto entry = CustomMessage::LoadVanillaMessageTableEntry(*textId);
        entry.msg = "My older brother will show you\x11the way, so follow him and\x11"
                    "don't get separated!";

        CustomMessage::LoadCustomMessageIntoFont(entry);
        *loadFromMessageTable = false;
```

Reply:
Now that Im home at my PC, I left this in case someone wanted to take a stab at actual replacing of multibox text they would already have a variable to test with.